### PR TITLE
Issue #5224: Fix typo in views build_form pager setting

### LINKED ISSUE #5224: Views: initial setting "Use a pager" ignored
**Description of the bug**

When creating an view, there's a checkbox "Use a pager", which is by default checked.
But when submitting that first form with "Continue and configure", this setting is ignored, the view gets created with "Display a specified number of items". No pager.

**Steps To Reproduce**

1. Go to admin/structure/views/add
2. Add a new view with "Use a pager" checked
3. Continue and configure
4. Verify that the pager setting was ignored

Other initial settings like "Create a menu link" or "Include an RSS feed" work as expected. And I also verified that this works as expected in Drupal 7 Views.

**Screenshots**

views-use-pager

views-no-pager

**Additional information**

- Backdrop CMS version: 1.20.0-preview

### PR DESCRIPTION
**Pull request by indigoxela**
Fixes https://github.com/backdrop/backdrop-issues/issues/5224

### CODE REVIEW

**Review by klonos — APPROVED** [review]
Good catch 👍🏼

File: core/modules/views_ui/wizards/views_ui_base_views_wizard.php [hunk 1]
```diff
@@ -153,7 +153,7 @@ function build_form($form, &$form_state) {
       '#min' => 0,
       '#step' => 1,
     );
-    $form['displays']['page']['options']['pagerz'] = array(
+    $form['displays']['page']['options']['pager'] = array(
```
Comment (klonos) [inline core/modules/views_ui/wizards/views_ui_base_views_wizard.php:156]: 😅

Comment (laryn) [reply core/modules/views_ui/wizards/views_ui_base_views_wizard.php:156]: @indigoxela continues to make the Backdropz better.

**Comment by laryn** [comment]
Good catch @indigoxela -- thanks for filing and fixing, and thanks @klonos for reviewing. I tested as well and it does what it should. :+1:  Merged into 1.x and 1.19.x.

**Pull request merged**